### PR TITLE
Ensure that dropdown does not move other elements

### PR DESCRIPTION
**Pull request by perok**
- [x] Fix missing shadows
image

Also fixes the following:
image
Now:
image

### CODE REVIEW

File: src/Nordea/Components/DropdownFilter.elm [hunk 1]
```diff
@@ -110,7 +110,8 @@ view (DropdownFilter options) attributes =
             , Css.border3 (Css.rem 0.0625) Css.solid Colors.grayMedium
             , Css.borderColor Colors.redDark |> styleIf (options.hasError || showHasNoMatch)
             , Css.borderRadius (Css.rem 0.25)
-            , Css.overflow Css.hidden
+            -- , Css.overflow Css.shown
```
Comment (perok) [inline src/Nordea/Components/DropdownFilter.elm:113]:
```suggestion
```

**Review by hungqt — APPROVED** [review]
🚀

File: src/Nordea/Components/DropdownFilter.elm [hunk 2]
```diff
@@ -175,7 +183,10 @@ inputSearchView hasFocus searchString onInput onFocus =
                     [ Css.outline Css.none
                     , Css.border3 (Css.px 1) Css.solid Colors.blueNordea
                     ]
-                , Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                , if hasFocus then
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                  else
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 4) (Css.px 4)
                 , Css.boxShadow5 Css.inset (Css.px 0) (Css.px -1) (Css.px 0) Colors.grayLight
```
Comment (perok) [inline src/Nordea/Components/DropdownFilter.elm:190]: Should this perhaps be removed? @TheaSommersletta 
image

File: src/Nordea/Components/DropdownFilter.elm [hunk 3]
```diff
@@ -175,7 +183,10 @@ inputSearchView hasFocus searchString onInput onFocus =
                     [ Css.outline Css.none
                     , Css.border3 (Css.px 1) Css.solid Colors.blueNordea
                     ]
-                , Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                , if hasFocus then
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                  else
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 4) (Css.px 4)
```
Comment (fredrikloberg) [inline src/Nordea/Components/DropdownFilter.elm:189]: Would be great to convert the px's to rem's, then everything scales accordingly 😀

File: src/Nordea/Components/DropdownFilter.elm [hunk 4]
```diff
@@ -175,7 +183,10 @@ inputSearchView hasFocus searchString onInput onFocus =
...
                 , Css.boxShadow5 Css.inset (Css.px 0) (Css.px -1) (Css.px 0) Colors.grayLight
                 , Css.height (Css.px 48)
                 --, Css.fontSize (Css.rem 1.0)
```
Comment (fredrikloberg) [inline src/Nordea/Components/DropdownFilter.elm:192]: (And the next line)
```suggestion
```

**Review by fredrikloberg — APPROVED** [review]
Nice 🤩

File: src/Nordea/Components/DropdownFilter.elm (hunk 3 above)
Comment (perok) [reply src/Nordea/Components/DropdownFilter.elm:193]: Good point! Everything here should probably be converted then

**Pull request merged**